op-node/rollup/finality/finalizer.go

```diff
@@ -313,10 +321,12 @@ func (fi *Finalizer) tryFinalize() {
 func (fi *Finalizer) findLastFinalizedL2BlockWithConsecutiveQuorom(
 	fdL2BlockNumber uint64, // candidate L2 block number to finalize
 	finalizedL2Number uint64, // last finalized L2 block number
+	gadgetActivatedTimestamp uint64, // BTC staking activated timestamp
```

Comment:
Can this be persisted somewhere in OP node?

Reply:
yes this will be stored using bbolt in the gadget. so whenever op call the API, it only reads from the db, not RPC calls

but luckily this is only implementation detail so once we make the changes, op-node won't even notice :)